express/src/routes/testing-routes.ts

```diff
@@ -58,20 +57,31 @@ router.post('/change-redirect-URIs/:serviceId/:selfServiceClientId/:clientId', a
         return;
     }
 
-    // further error handling around a valid array of valid URIs
-
     const redirectUris = req.body.redirectURIs.split(" ");
+    for (let i = 0; i < redirectUris.length; i++) {
```

Comment:
As discussed this should be middleware that's inserted into the route rather than inline code.  As middleware it can be re-used for the other URI client property.